crates/rooch-server/src/jsonrpc_types/rooch_types.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use rooch_types::transaction::{TransactionType, TypedTransaction};
+
+use moveos_types::transaction::MoveAction;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct TransactionView {
```

Comment:
We may need to put the original transaction bytes in the View. Convenient for the client to parse the Transaction.